ui/packages/catalog-ui-search/src/main/webapp/component/input/boolean/input-boolean.presentation.tsx

```diff
@@ -0,0 +1,44 @@
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser
```

Comment:
✏️ This file belongs in the react-component directory

Reply:
Done